maven-artifact/src/test/java/org/apache/maven/artifact/versioning/DefaultArtifactVersionTest.java

```diff
@@ -80,6 +80,15 @@ public void testVersionParsing() {
         checkVersionParsing("1.2.3-200705301630", 1, 2, 3, 0, "200705301630");
     }
 
+    public void testVersionParsingNot09() {
+        String ver = "१.२.३";
+        assertTrue(Character.isDigit(ver.charAt(0)));
+        assertTrue(Character.isDigit(ver.charAt(2)));
+        assertTrue(Character.isDigit(ver.charAt(4)));
+        ArtifactVersion version = newArtifactVersion(ver);
+        assertEquals(ver, version.getQualifier());
+    }
```

Comment:
Hold on, shouldn't this fail?

Reply:
No, as these are not numbers anymore, they are qualifiers (otherwise it would go into major, minor...)

Reply:
man, I want `U+1F4A9` as qualifier!

Reply:
but we must carry thru same change on Resolver as well...